.github/workflows/yaml-lint.yaml

```diff
@@ -0,0 +1,18 @@
+name: Yamllint Github Actions
+on: 
+  pull_request:
+    branches: 
+      - master
+
+jobs: 
+  lintFalcoRules: 
+    name: Yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3 
+      - name: checkout falco-rules
+        uses: ibiqlik/action-yamllint@v3
+        with: 
+          files: /rules/*.yaml 
+          yamllint_strict: true
+          yamllint_comment: true
```

Comment:
I can't find these 3 input parameters in the yamllint action: https://github.com/ibiqlik/action-yamllint?tab=readme-ov-file#optional-input-parameters

Reply:
yamllint_strict: true
          yamllint_comment: true
 @FedeDP  by default these two options have `false` value. Am not able to figure out in this  https://github.com/ibiqlik/action-yamllint?tab=readme-ov-file#optional-input-parameters what value is set, So just to be safe i set these. If you think we should go with default values then we can remove it.

Reply:
I mean, they are not existent input keys, right? We have:
```

    config_file - Path to custom configuration
    config_data - Custom configuration (as YAML source)
    file_or_dir - Enter file/folder (space separated), wildcards accepted. Examples:
        . - run against all yaml files in a directory recursively (default)
        file1.yaml
        file1.yaml file2.yaml
        kustomize/**/*.yaml mychart/*values.yaml
    format - Format for parsing output [parsable,standard,colored,github,auto] (default: parsable)
    strict - Return non-zero exit code on warnings as well as errors [true,false] (default: false)
    no_warnings - Output only error level problems [true,false] (default: false)

```

Reply:
Yes @FedeDP they aren't existing input keys. I think we should remove these two points ?
```
      yamllint_strict: true
      yamllint_comment: true
```